example/tools/pom.xml

```diff
@@ -31,6 +31,13 @@
      
   <name>${project.groupId}:${project.artifactId}</name>
 
+  <properties>
+    <apache.beam.groupId>cz.o2.proxima.beam</apache.beam.groupId>
+    <apache.beam.version>${project.version}</apache.beam.version>
+    <apache.beam.version.raw>2.18.0-SNAPSHOT</apache.beam.version.raw>
```

Comment:
I think this should be inherited from parent. We can move this from `beam/pom.xml` to root `pom.xml`.

Reply:
Moved.